substrate/client/network/types/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+description = "Substrate network types"
+name = "sc-network-types"
+version = "0.10.0-dev"
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+authors.workspace = true
+edition.workspace = true
+homepage = "https://substrate.io"
+repository.workspace = true
+documentation = "https://docs.rs/sc-network-types"
+
+[dependencies]
+bs58 = "0.4.0"
+libp2p-identity = { version = "0.1.3", features = ["ed25519", "peerid"] }
+litep2p = { git = "https://github.com/altonen/litep2p", branch = "master" }
```

Comment:
should we have a release before merging?

Reply:
Yes